code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -380,9 +380,9 @@ Bonus
 	switch(M.radiation)
 		if(0)
 			return FALSE
-		if(1 to 350)
+		if(1 to RAD_MOB_SAFE)
 			return 0.25
-		if(351 to 750)
+		if(RAD_MOB_SAFE + 1 to 750)

```

Comment:
you don't need the +1

Reply:
or to be accurate, with it decimal radiation values between RAD_MOB_SAFE and RAD_MOB_SAFE + 1 will trigger the else and give 1.5